sovryn-token-bridge/bridge/contracts/Federation.sol

```diff
@@ -1,7 +1,9 @@
 pragma solidity ^0.5.0;
```

Comment:
As a quick note, there seems to be a weird versionin scheme in this repo, where the newest version of a contract is always `Contractname.sol` and the previous contract is copied to `Contractname_vN.sol` where N is a running number. So perhaps the old federation contract should be copied under `Federation_v4.sol`

Reply:
@bravepooh what do you think ?